composer.json

```diff
@@ -29,26 +29,23 @@
         "php": ">=7.4",
         "ext-json": "*",
         "keboola/storage-api-client": "^13.0|^14.0",
-        "symfony/http-foundation": "^4.4|^5.2",
-        "symfony/validator": "^4.4|^5.2"
+        "symfony/http-foundation": "^4.4|^5.2|^6.0",
+        "symfony/validator": "^4.4|^5.2|^6.0"
     },
     "require-dev": {
-        "keboola/coding-standard": "^13.0",
-        "php-parallel-lint/php-parallel-lint": "^1.3",
+        "keboola/coding-standard": ">=13.0",
         "phpstan/phpstan": "^1.5",
         "phpstan/phpstan-phpunit": "^1.1",
         "phpunit/phpunit": "^9.5",
-        "symfony/dotenv": "^5.4"
+        "symfony/dotenv": "^4.4|^5.2|^6.0"

```

Comment:
Proc u error libky si odstranoval 4.4 a tady zustala?

Reply:
protože jsem ještě nebyl tak odvážnej :D dám to pryč